FreeRTOS/FreeRTOS-Plus-TCP
# Add required define to enable st driver

Description
-----------
When adding the port for the STM32F7 driver, you need to define that the module is enabled.  The define might be automatically set when using the CUBE IDE but it is not if you're using the SDK directly, like from https://github.com/STMicroelectronics/STM32CubeF7

Test Steps
-----------
Try to add port, and you will get zero symbols out of the driver, in which case NetworkInterface.o cannot find required HAL symbols.

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
